src/Screens/Bookmark/BookmarkScreen.js

```diff
@@ -1,14 +1,109 @@
-import { View, Text } from "react-native";
 import React from "react";
+import { View, Text, FlatList, Pressable } from "react-native";
+import styles from "../../Shared/styles";
+import { useSelector } from "react-redux";
+import { useNavigation } from "@react-navigation/native";
+import Icon from "../../Shared/Components/Icon.js";
+import { RoutingItems } from "../../Data/RoutingLookups"; //to determine which screen the legislation is to navigate to.
 import Breadcrumb from "../../Shared/Components/header/Breadcrumb";
 
 const BookmarkScreen = () => {
-  return (
+  const navAid = useNavigation();
+  const bookmarks = useSelector(state => state.bookmarks); //get bookmarks array
+
+  const renderBookmarkItem = ({ item }) => (
```

Comment:
I'm just putting this out there; but in the effort to make smaller more digestable code, couldn't this be a separate bookmark card component?

Also, the styling of this does not match the XD; not sure if that's just on my screen?

Reply:
Okay I have moved the display code to a "card" component.  And fixed the styling to match the Search container